components/raftstore/src/store/peer.rs

```diff
@@ -2103,7 +2129,8 @@ where
                 }
                 self.should_wake_up = true;
                 let p = Proposal {
-                    is_conf_change: req_admin_cmd_type == Some(AdminCmdType::ChangePeer),
+                    is_conf_change: req_admin_cmd_type == Some(AdminCmdType::ChangePeer)
+                        || req_admin_cmd_type == Some(AdminCmdType::ChangePeerV2),
```

Comment:
Is it a bug?

Reply:
Not, it is a mistake but not a bug, because we use `Callback::None` as the callbacks for `ChangePeer` cmds and we will not send these callbacks to the apply thread. Also, if these callbacks did send to the apply thread, some test cases will not pass.

https://github.com/tikv/tikv/blob/9b8785a42f6e03621c0d0ce66799761b3e00f9cc/components/raftstore/src/store/worker/pd.rs#L858

https://github.com/tikv/tikv/blob/9b8785a42f6e03621c0d0ce66799761b3e00f9cc/components/raftstore/src/store/peer.rs#L113-L115